### ISSUE TITLE
When will you release a new model?

### ISSUE THREAD

**Issue by young-chao** [SLAM-group/newhope#1]
I am very interested in the newhope model, because it still performs well in some more complicated code tasks. I believe it uses some very high -quality data sets and training techniques to achieve the current performance. 
Can I expect this model to be released in a new version after cleaning up the data set? If so, how long will it take? I also look forward to the release of related papers and documents. 
I earnestly look forward to your reply, thank you.

**Comment by Nurb432** [comment]
Agreed, the one that was here was great for coding and was happy for it to be here. Will you guys be coming back, or is it over?

**Comment by young-chao** [comment]
你好，你的邮件已收到，谢谢！

**Comment by BigGunLive** [comment]
I also think he performed better than I expected
